code/game/machinery/machinery.dm

```diff
@@ -466,10 +466,8 @@ Class Procs:
 /obj/machinery/tesla_act(var/power)
 	..()
 	if(prob(85))
-		emp_act(2)
+		explosion(src.loc,1,2,4,flame_range = 2, adminlog = 0, smoke = 0)
```

Comment:
Like I said before, this should be the LEAST common effect, not the most. This makes the tesla absolutely rip the station in half, like what the hell.